node/utils/tests/files.rs

```diff
@@ -116,3 +116,23 @@ fn read_from_toml() {
     assert_eq!(config.keys.github, "xxxxxxxxxxxxxxxxx");
     assert_eq!(config.keys.travis.as_ref().unwrap(), "yyyyyyyyyyyyyyyyy");
 }
+
+#[test]
+fn count_files_in_a_dir() {
+    let msg = "Hello World!";
+    let path = "./test_string_read_file/";
+    let file_name = "out.keystore";
+    if let Err(e) = write_to_file(&msg.as_bytes().to_vec(), &path, file_name) {
+        panic!(e);
+    }
```

Comment:
Can you change this to:

```
    write_to_file(msg.as_bytes(), &path, file_name).unwrap();
```

Sorry that it's set up wrong above in this file, I can make a pass through after to fix, but there is no need to heap allocate the message or manually handle the error in this test

Reply:
ok